pkgs/by-name/gr/grmon/package.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, nix-update-script
+}:
+
+buildGoModule rec {
+  pname = "grmon";
+  version = "0.1";
+
+  src = fetchFromGitHub {
+    owner = "bcicen";
+    repo = "grmon";
+    rev = "v${version}";
+    hash = "sha256-0J7f4DMADUut3Da0F1eTDsT1Hlk0rfInwzbcVcQNzg8=";
+  };
+
+  vendorHash = "sha256-ySgWEGHlEJpfB/BZuRs1bELBspEaiaX/UnJai2V/hx0=";
+
+  CGO_ENABLED = "0";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-buildid="
+  ];
+
+  passthru = {
+    updateScript = nix-update-script { };
+  };
+
+  meta = with lib; {
+    description = "Command line monitoring for goroutines";
+    longDescription = ''
+      grmon allows interactive monitoring of goroutines. To use it, instrument your Go code following
+      the [usage description of the project](https://github.com/bcicen/grmon?tab=readme-ov-file#usage).
```

```suggestion
      To use it, instrument your Go code following the usage description of the project at https://github.com/bcicen/grmon?tab=readme-ov-file#usage.
```

We shouldn't duplicate description. Also I don't think links work here, yet.

---

Links should work, long description is markdown: https://nixos.org/manual/nixpkgs/unstable/#var-meta-longDescription

---

I couldn't find an example on search.nixos.org where that works but let's give it a try

---

Found a proof that markdown works ^ ^ https://github.com/NixOS/nixpkgs/pull/280291